lib/src/client_task_impl.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:meilisearch/src/client_impl.dart';
+import 'package:meilisearch/src/update_status.dart';
+
+import 'pending_update.dart';
+
+class ClientTaskImpl implements PendingUpdate {
+  final int updateId;
+  final MeiliSearchClientImpl client;
+
+  ClientTaskImpl(this.client, this.updateId);
+
+  factory ClientTaskImpl.fromMap(
+    MeiliSearchClientImpl client,
+    Map<String, dynamic> map,
+  ) =>
+      ClientTaskImpl(client, map['uid'] as int);
+
+  @override
+  Future<UpdateStatus> getStatus() async {
```

Comment:
Did this method will become `getTask` ?

Reply:
I thought the name will be changed in the next PR but I think this `getStatus` method and `getUpdateStatus` should be renamed in `getTask` and `getTasks`.

Reply:
Yeah, I just tried to reduce the scope of changes haha, but I'll do that, thanks for the reminder ;)